### ISSUE TITLE
Location by Layer

### ISSUE THREAD

**Issue by ludovico85** [3liz/lizmap-web-client#715]
Hi,
the function returns a completely wrong location. I have read that is an issue related with the 2.16 Qgis server version but I'am using the 2.18 version (The server runs on ubuntu xenial). I have restricedt the project to the following SR:
ESPG 32633 - 4326 - 3857

Thanks

**Comment by mdouchin** [comment]
In Lizmap master (future 3.2), there is an option in the Admin panel config page to tell Lizmap that QGIS Server is >= 2.18. 
Since 2.18, the data sent by QGIS server for WFS GetFeature requests (used for attribute table, location by layer and other tools) is not in the layer projection anymore, but follows the GeoJSON standard EPSG:4326

**Comment by ludovico85** [comment]
Thank you, now it works!

**Comment by rldhont** [comment]
This option has been backported to 3.0 and 3.1.

**Comment by aeduard** [comment]
Hi! I'm using Lizmap 3.1, the method described by **_mdouchin_** worked, but there is a small glitch. Clicking the zoom button from the pop-up of a point feature moves the map close to the origin of the coordinate system. In my case ESPG 31700 (Romania), and the map moves to the Adriatic Sea near Albania. If I'm clicking to the zoom or move button in the data table, the function works OK. Strangely after a click here (data table), the zoom is OK also from the pop-up.

**Comment by rldhont** [comment]
@aeduard do you updated lwc and tested the issue ?

**Comment by aeduard** [comment]
No, I didn't update the client, the bug is still there. So far is not so important to make the update.
